ngraph/test/CMakeLists.txt

```diff
@@ -642,6 +642,11 @@ if (NGRAPH_ONNX_IMPORT_ENABLE)
     target_include_directories(unit-test PRIVATE ${ONNX_IMPORTER_SRC_DIR}/src)
 
     target_link_libraries(unit-test PRIVATE onnx_importer)
+    if (LINUX)
+        target_link_options(unit-test PRIVATE -Wl,--exclude-libs,ALL)
```

Comment:
Are these flags correct for LINUX compilation via clang?

Reply:
yes